src/python/deepgnn/graph_engine/snark/server.py

```diff
@@ -43,38 +43,55 @@ def __init__(
         """Create server and start it.
 
         Args:
-            path (str): location of graph binary files. If given hdfs:// or adl:// path see config_path and stream parameters,
-                adl://name.azuredatalakestore.net/path/to
-                hdfs://localhost:9000/path/to
-            partitions (List[int]): which partitions to load.
+            meta_path: location of meta.txt file with global graph information. If given hdfs:// or adl:// path
+                use config_path and stream parameters, for additional configuration.
+            partitions (Sequence[Union[int,Tuple[str, int]]]): Partition ids to load from meta_path folder.
+                List can contain a tuple of path and partition ids to load to override location of a binary file for given partition.
+                For additional path configuration see `meta_path` parameter.
             hostname (str): address to use for serving.
             ssl_config (Dict[str, str], optional): SSL configs for encrypted connections with clients if needed. Defaults to None.
             storage_type (PartitionStorageType, default=memory): What type of feature / index storage to use in GE.
-            config_path (str, optional): Path to folder with configuration files.
+            config_path (str, optional): Path to folder with configuration files for hdfs access.
             stream (bool, default=False): If remote path is given: by default, download files first then load,
                 if stream = True and libhdfs present, stream data directly to memory.
         """
+        if partitions is None:
+            partitions = [(meta_path, 0)]
+        partitions_with_path: List[Tuple[str, int]] = []
+        for p in partitions:
+            if isinstance(p, int):
+                partitions_with_path.append((meta_path, p))
+            elif len(p) == 2:
```

Comment:
Since we now support the old MemoryGraph(data_dir, [0]) can we revert all the tests and docs to use the original format.

Reply:
Why? I'd rather be more explicit.